idaes/gas_solid_contactors/unit_models/bubbling_fluidized_bed.py

```diff
@@ -2087,9 +2249,79 @@ def initialize(blk, gas_phase_state_args=None, solid_phase_state_args=None,
                         idaeslog.condition(results))
                         )
         else:
-            init_log.warning('{} Initialisation Step 6 Failed.'
+            init_log.warning('{} Initialization Step 6 Failed.'
                              .format(blk.name))
```

Comment:
We have a new standard where initialization routines should raise an `InitializationError` (new error type) if the final step fails. This is to allow other models/flowsheets to be aware when a unit model (or property model, etc.) fails to initialize properly (which used to pass quietly in the background).

Reply:
> I can't speak to the details (not enough time to go through in detail and too long since I worked on this model), but I did have a couple of comments on new IDAES standards we should include in this PR if possible (they should be easy changes).
> 
> My only other comment is that I was surprised to see changes in the model results for something that apparently only changes scaling and units definitions. Is this due to changes in the units for certain variable, or did something more significant change?

I think the most significant changes to the model results are because of the changes to the value of the pressure drop across the orifice. The correct value to use 3.4E3 Pa.